src/frontend/src/icp/components/send/IcSendTokenWizard.svelte

```diff
@@ -190,6 +191,12 @@
 		store: initEthereumFeeStore()
 	});
 
+	const { store: ethereumFeeStore } = getContext<EthereumFeeContext>(ETHEREUM_FEE_CONTEXT_KEY);
+
+	// In case we are converting ERC20 to ckERC20, we need to include ckETH fees in the transaction.
+	let ckEthMaxTransactionFee: bigint | undefined = undefined;
+	$: ckEthMaxTransactionFee = $ethereumFeeStore?.maxTransactionFee;
```

Comment:
@peterpeterparker the doubt I asked you about is:

this value is updated by the child `EthereumFeeContext`, so, will it be fetched here (in the parent) correctly?